Speaker Feedback: Show both form and approved feedback to organizers on the front end

coreymckrill opened the pull request:
Organizers need to be able to see what a speaker would see when viewing their approved feedback, especially so the view can be styled to match the rest of the site. But organizers should also be able to submit feedback about a session through the form, just like everyone
else. So this makes it so everyone except the session speaker sees the form, and both the session speaker and organizers see the approved feedback list.

This also hides the "helpful" button on approved feedbacks from organizers. Only the speakers should be able to mark something as helpful.

Fixes #457

### How to test the changes in this Pull Request:

1. View the feedback URL for a session while logged out. Only the feedback form should be visible.
1. View the feedback URL for a session while logged in as a session speaker. Only the list of approved feedback should be visible.
1. View the feedback URL for a session while logged in as a site admin. Both the feedback form and the list of approved feedback should be visible. However, the "Was this feedback helpful?" section should be omitted from each approved feedback.

coreymckrill committed:
Remove unused import

coreymckrill committed:
Speaker Feedback: Show both form and approved feedback to organizers

Organizers need to be able to see what a speaker would see when
viewing their approved feedback, especially so the view can be styled
to match the rest of the site. But organizers should also be able to
submit feedback about a session through the form, just like everyone
else. So this makes it so everyone except the session speaker sees the
form, and both the session speaker and organizers see the approved
feedback list.

This also hides the "helpful" button on approved feedbacks from
organizers. Only the speakers should be able to mark something as
helpful.

Fixes #457

ryelle commented on public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/class-walker-feedback.php:
```diff
@@ -96,20 +97,22 @@ protected function render_feedback( $comment, $depth, $args ) {
-						<?php esc_html_e( 'Yes', 'wordcamporg' ); ?>
-					</button>
-				</footer>
+				<?php if ( in_array( get_current_user_id(), $session_speakers, true ) ) : ?>
```
I think this should still be displayed, but it should be un-clickable, just in case anyone wants to style it (also so organizers know it should be there). I think if you remove the input when not a speaker, that should prevent the submission.

maybe with a note … somewhere… that only speakers can mark things as helpful?

coreymckrill replied:
Fair enough. Fixed here: c16c8302

ryelle reviewed (commented):
Working great, just the one comment about the helpful UI.

coreymckrill committed:
Disable "helpful" input for organizers instead of removing it

The helpful toggle should still be visible to organizers in case they
want to style it (and just so they know it's there). But it shouldn't
be togglable for them. This also adds a notice at the top of the list
of the feedback that only organizers will see that explains that they
can't use the toggles.
